ex_2.js

```diff
@@ -0,0 +1,38 @@
+// Exercitiu 2: Scrieți o funcție care returnează un obiect nou fără proprietățile furnizate
+
+//Funcția primește 2 parametri ca date de intrare. Unul dintre parametri este obiectul, iar celălalt este proprietatea care nu se dorește să persiste în rezultatul returnat.
+
+//Exemplu 1:
+
+//Input: obiect = { a: 1, b: 2 }, proprietate = “b”
+
+//Output: { a: 1 }
+//
+
+
+function removeProperties(car, model) {
+
+    const newcar = { ...car };
+
+    model.forEach(model => {
```

Comment:
```suggestion
    proprietes.forEach(property => {
```

Daca numim argumentul `properties` in loc de `model` atunci si in functia foreach - ii dam aceasta denumire. Oare nu arata mai sugestiv astfel?